cpg_workflows/stages/gcnv.py

```diff
@@ -313,10 +313,17 @@ def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput:
         germline_calls = inputs.as_dict_by_target(GermlineCNVCalls)
         jobs = []
         for sgid, new_vcf in expected.items():
-            if sgid == 'placeholder':
+            if sgid == 'placeholder' or sgid not in cohort.get_sequencing_group_ids():
```

Comment:
Is the call to `cohort.get_sequencing_group_ids()` cached? If not, this could be slow as it executes for each sg in the loop. Might be better to take it out of the for loop in this case.

Reply:
nice spot. In cpg-flow it will be a dict, but for now it's not so efficient. I've updated it!